app/controllers/outgoing_text_messages_controller.rb

```diff
@@ -1,8 +1,30 @@
 class OutgoingTextMessagesController < ApplicationController
   include TwilioRequestable
+  process_twilio_actions only: [:update]
+  before_action :redirect_to_client_unless_admin, only: :create
+
+  def create
+    message = OutgoingTextMessage.new(outgoing_text_message_params)
+    if message.save
+      SendOutgoingTextMessageJob.perform_later(message.id)
+    end
+    redirect_to client_path(id: outgoing_text_message_params[:client_id])
+  end
 
   def update
```

Comment:
This is a duplicate of `text_status_callback` - if we're refactoring, can we pick one and remove the other?